### PR TITLE
Added  Wechaty token section

### PR DESCRIPTION
**Pull request by shraddhavp**
This PR is for the Wechaty token section, this links #890

### CODE REVIEW

**Comment by shraddhavp** [comment]
@huan  @Rohitesh-Kumar-Jain  and @proudofsimin please check out the PR

File: docusaurus/docs/specs/specs-token.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+title: Wechaty Puppet Service Token Gateway
+sidebar_label: Token
+---
+
+## What is Wechaty Puppet Service Token Gateway?
+
+Wechaty Puppet Service Token Gateway is a gateway for converting the Wechaty Puppet Provider to a Wechaty Puppet Service.This can be done by setting up different environment variables
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: add a space between two sentences and a period at the end of the second sentence.

File: docusaurus/docs/specs/specs-token.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Here 👇 is an example for setting up Wechaty Puppet service to one of the Wechaty Puppet Provider Padlocal
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: I don't quite understand this part. By example you mean the following four variables (WECHATY_PUPPET, WECHATY_TOKEN...)? And I could see you are trying to create a casual atmosphere, but I felt the icon makes this document a bit unprofessional. I would recommend to remove it.

File: docusaurus/docs/specs/specs-token.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+## WECHATY_PUPPET
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: If these four are the examples mentioned in the previous sentence, they should be one level lower.

File: docusaurus/docs/specs/specs-token.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+This environment variable is used for defining the Wechaty Puppet Provider NPM name and its parameters
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: Don't forget the period at the end.

File: docusaurus/docs/specs/specs-token.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  -e WECHATY_TOKEN \
+  -e WECHATY_PUPPET_SERVER_PORT \
+  \
+  -e WECHATY_LOG \
+  \
+  -p "$WECHATY_PUPPET_SERVER_PORT:$WECHATY_PUPPET_SERVER_PORT” \
+  wechaty/wechaty:0.56
+```
+
+The after running the above commands lot of  output log messages can be seen in the terminal.
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: I don't understand this sentence.

File: docusaurus/docs/specs/specs-token.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+
+## Service Discovery
+
+The running of the Wechaty Puppet Service can be checked by using the command given below:
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: Use the command given below to check the running of the Wechaty Puppet Service.
(Active voice is preferred)

File: docusaurus/docs/specs/specs-token.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+```sh
+curl https://api.chatie.io/v0/hosties/${WECHATY_TOKEN}
+```
+
+**Replace ${WECHATY_TOKEN} to your real token in the above configuration*
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: Are you trying to make it **bold** ?

**Review by proudofsimin — CHANGES_REQUESTED** [review]
I'm feeling like you've gone too far in this part. Hold on, and don't give instructions in reference. The most important things in reference are easy to search and include the information as much as possible, so the word like "example" should not appear as you are supposed to provide everything like an encyclopedia. I found a good [example](https://developers.google.com/vault/reference/rest) for your reference. I think you may consider reconstruct this part a little bit. Please let me know if you have any question.

**Comment by shraddhavp** [comment]
> I'm feeling like you've gone too far in this part. Hold on, and don't give instructions in reference. The most important things in reference are easy to search and include the information as much as possible, so the word like "example" should not appear as you are supposed to provide everything like an encyclopedia. I found a good [example](https://developers.google.com/vault/reference/rest) for your reference. I think you may consider reconstruct this part a little bit. Please let me know if you have any question.

ok will do the changes @proudofsimin ,thanks

**Comment by shraddhavp** [comment]
@proudofsimin  have done the changes please check now,let me know if any further changes

File: docusaurus/docs/specs/specs-token.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+| WECHATY_PUPPET_SERVER_PORT | sets up the  free server port for the Wechaty Puppet Service,also used for docker port mapping                                                                                                                                                                                                           | ```bash export WECHATY_PUPPET_SERVER_PORT=8788 // any available port can be visited from internet```    |
+| WECHATY_LOG                | sets up the log mode for the service. Usually initialize the variable to `Verbose` mode as this gives more debug log messages. `verbose` mode is an option available in many OS that gives details on what the computer is doing,which drivers and software are being installed or loaded and many more. | ```bash export WECHATY_LOG="verbose"```                                                                 |
+
+## Starting Wechaty Puppet Servcie Token Gateway
+
+After the docker image has been created use a docker command  to start the Wechaty Puppet Service Token Gateway with the below configuration:
```
Comment (proudofsimin) [inline docusaurus/docs/specs/specs-token.md]: After the docker image has been created [,] use a docker command  to start the Wechaty Puppet Service Token Gateway with the below configuration:

(Don't forget the comma)

**Review by proudofsimin — APPROVED** [review]
Thank you! The updated version is impressive :)

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
I am approving for PR quality 

Perfect PR : clean commit history, properly based, PR good in scope.

**Comment by lijiarui** [comment]
hi @shraddhavp  As GSoD Project PR Review Rules in https://github.com/wechaty/wechaty.js.org/issues/1045

This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.

**Comment by Rohitesh-Kumar-Jain** [comment]
@Soumi7 We would be glad if you may once go through the PR, and either request for changes or approve the PR.

**Review by Soumi7 — APPROVED** [review]
LGTM

**Comment by Rohitesh-Kumar-Jain** [comment]
> LGTM

Welcome back, glad to hear again from you 🎉

**Comment by Soumi7** [comment]
> > LGTM
> 
> Welcome back, glad to hear again from you tada

Haha Thanks! Great to be back!

**Review by huan — APPROVED** [review]
Thanks for adding this!

Please change the file name to `specs/gateway.mdx` as other files are.

After being renamed, I believe this PR will be ready to be merged.

File: docusaurus/docs/specs/specs-token.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+  \
+  -e WECHATY_PUPPET \
+  -e WECHATY_PUPPET_PADLOCAL_TOKEN \
+  \
+  -e WECHATY_TOKEN \
+  -e WECHATY_PUPPET_SERVER_PORT \
+  \
+  -e WECHATY_LOG \
+  \
+  -p "$WECHATY_PUPPET_SERVER_PORT:$WECHATY_PUPPET_SERVER_PORT” \
```
Comment (huan) [inline docusaurus/docs/specs/specs-token.md]: Please add `-e WECHATY_PUPPET_SERVICE_NO_TLS_INSECURE_SERVER`

See: https://github.com/wechaty/wechaty-puppet-service/issues/160

**Pull request merged**